bin/minifollowups/pycbc_injection_minifollowup

```diff
@@ -140,6 +144,49 @@ except KeyError:
     dec_chirp_dist = pycbc.pnutils.chirp_distance(dec_dist, mchirp)
     sorting = dec_chirp_dist.argsort()  # ascending order of dec chirp distance
 
+# Get the trigger SNRs and times
+# But only ones which are within a small window of the missed injection
+missed_inj_times = numpy.sort(f['injections/tc'][:][missed])
+
+# Note: Adding 2 * Earth diameter in light seconds to the window here
```

Comment:
Should it be 2 diameters or just 1, given that you have a left and right edge? 
In fact, if the injection stated end time is at geocentre, should it not rather be Earth radius? 

I guess if we're not totally clear about the geocentre then 1* diameter on each side is 'safe'

Reply:
I have updated to 1 diameter